### ISSUE TITLE
Temporary files in assembly stage

### ISSUE THREAD

**Issue by fredchaux** [fenderglass/ABruijn#549]
Dear Flyers :)
First of all, wonderful tool !
I am trying to assemble de novo a large genome (unknown, probably >1Gb, + bacterial community) using long Nanopore reads (Super accuracy, around 30 Gb), but it crashes during assembly stage (index filling most often). Is there a possibility to get temporary files from the k-mer counting and/or index filling steps? That would be helpful to resume computation from these points. Also, is there a (empirical) formula to estimate a priori de RAM required?
May thanks in advance,
Fred

**Comment by mikolmogorov** [comment]
Hi Fred,

Sorry for the lates response! Flye will save info between major stages (e.g. disjointig assembly). Indexing step is typically much shorter than the rest of disjointig assembly, so there is no functionality to save indexing results. There is also no exact formula, because memory usage could be different for different organisms (with various repeat content and heterozygosity). I'd say you may need about 100Gb of RAM for your dataset. Make sure you are using `--nano-hq`, it has better memory usage.

Best,
Mikhail

**Comment by fredchaux** [comment]
Hi Mikhail,
Many thanks for taking the time for answer. Indeed I increased time and it
worked (peak usage 102 GB).
By the way, I did not find in the manual how flye works with telomeres. Is
there a way to explicitly state their sequence (eg TTAGGGG) when it s
different from mammals? I guess that would help both the assembler itslef
and end users, isn't it?
Great algorithm anyway,
Best regards,
Free Chaux

Le lundi 28 novembre 2022, Mikhail Kolmogorov ***@***.***> a
écrit :
